examples/tutorials/advance_model_config/02_device.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<blockquote>Next, batch data is split across available GPUs and gradients are computed separately and then averaged on one device (usually on the first GPU of the available).</blockquote><p>may be is splited? or splitting?</p><p>Generally, this sentence is difficult to perceive, it is better to divide it into two different ones.</p><p> </p><p> </p><blockquote>Initialization of a large model on a large number of GPUs may take some time (minutes or tens of minutes)!</blockquote><p>use synonym of large</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>